student.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'person'
+
+# The Student class represents a student with an id, name, age, and classroom.
+# It inherits from the Person class and includes an additional method to play hooky.
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(age, classroom, name = 'Unknown', parent_permission: true)
+    super(name, age, parent_permission)
```

Comment:
- [ ] Please make sure to pass the parameters in the super method in the correct order as the parent class initializer expects.
  - You can consider the below code.
  
```ruby
super(age, name, parent_permission)